### ISSUE TITLE
Failed to activate the apathy-theme theme

### ISSUE THREAD

**Issue by dimonge** [coopermaruyama/apathy-theme#28]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.7.2
**System**: Microsoft Windows 10 Pro
**Thrown From**: [apathy-theme](https://github.com/coopermaruyama/apathy-theme) package, v1.10.4
### Stack Trace

Failed to activate the apathy-theme theme

```
At 'languages/jsx.less' wasn't found in C:\Users\peter.shodeinde\.atom\packages\apathy-theme\styles\base.less:403:0

LessError: 'languages/jsx.less' wasn't found
  at C:\Users\peter.shodeinde\.atom\packages\apathy-theme\styles\base.less:403:0
```
### Commands

```
     -1:13.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "closeEmptyWindows": false,
    "destroyEmptyPanes": false,
    "excludeVcsIgnoredPaths": false,
    "openEmptyEditorOnStart": false,
    "themes": [
      "seti-ui",
      "apathy-theme"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
nuclide, vundefined
react, v0.14.1
seti-ui, v0.9.2

# Dev
No dev packages
```

**Comment by gvagenas** [comment]
Same problem on Linux Mint 17.3 while for OSx ElCapitan is working fine

**Comment by ibraheeamm** [comment]
Same problem with osx elcapitan here

**Comment by alice-em** [comment]
+1 OSX 10.11.15

**Comment by czxtm** [comment]
whoops! forgot to check-in a new file. Fixed in 1.10.5.

By the way, atom dropped support for a lot of things this syntax theme depended on, which is why it only runs correctly in dev mode. If you just want the styles, I released them here: https://atom.io/themes/vivid
